ebml_matroska.xml

```diff
@@ -882,7 +882,9 @@ redundant framing information while preserving versioning and semantics between
 
 Value represents a clockwise rotation, in degrees, around the up vector. This rotation must be applied
 before any `ProjectionPosePitch` or `ProjectionPoseRoll` rotations.
-The value of this field should be in the -180 to 180 degree range.</documentation>
+The value of this field should be in the -180 to 180 degree range.
+
+Setting `ProjectionPoseYaw` to 180 or -180 degrees, with the `ProjectionPoseYaw` and `ProjectionPosePitch` set to 0 degrees flips the image horizontally.</documentation>
```

Comment:
One of the two `ProjectionPoseYaw` should be `ProjectionPoseRoll`, doesn't it?

Reply:
yes, good catch

Reply:
Fixed